cmd/access/node_builder/access_node_builder.go

```diff
@@ -262,6 +267,8 @@ func DefaultAccessNodeConfig() *AccessNodeConfig {
 		registerCacheType:                    pStorage.CacheTypeTwoQueue.String(),
 		registerCacheSize:                    0,
 		programCacheSize:                     0,
+		versionControlEnabled:                true,
+		stopControlEnabled:                   true,
```

Comment:
this should default to `false` since operators need to take manual action for the stop control to work